modelservice/games/scopes/base.py

```diff
@@ -305,22 +305,29 @@ def _scope_tree(self, *args, **kwargs):
         scope_groups = self.child_scopes
         for resource_name, scope_group in scope_groups.items():
             self.log.debug(
-                '_scope_tree: resource_name: {name}, user: {user!s}',
-                name=resource_name, user=user)
+                '_scope_tree: resource_name: {name}, user: {user!s}, exclude: {exclude}',
+                name=resource_name, user=user, exclude=exclude)
 
-            children = scope_group.for_user(user)
+            if resource_name == exclude:
```

Comment:
I wonder is `exclude` should be a list instead of a string, so we could allow to exclude more than one type of resource with `if resource_name in exclude`

Reply:
I like that and will incorporate it.

Reply:
Done.